tests/test_pypuck.py

```diff
@@ -1 +1,65 @@
+# authors: Jarvis Nederlof
+# date: 2020-03-02
+
+"""
+This script tests the pypuck functions in the pypuck module.
+"""
+
 from pypuck import pypuck
+import pandas as pd
+
+# @pytest.mark.skip()
+def test_player_stats_good(start_date='2019-10-02', end_date='2020-02-28'):
+	"""
+	Test function to check proper inputs and returns.
+
+	Keyword Arguments:
+		start_date {str} -- start_date to query (default: {'2019-10-02'})
+		end_date {str} -- end_date to query (default: {'2020-02-28'})
+
+	Raises:
+		TypeError: A message if return type is wrong (wrong data type).
+		ValueError: A message if return value is wrong (not enough data).
+	"""
+	# Test for various end_dates
+	for end_date in [end_date, None]:
+		df = pypuck.player_stats(start_date, end_date)
+		if isinstance(df, pd.DataFrame) is False:
+			raise TypeError("player_stats() didn't return a pd.DataFrame.")
+
+		if len(df) != 100:
+			raise ValueError("player_stats() didn't return the top 100 players.")
+
+	# Test for various start_dates
+	for start_date in [start_date, None]:
+		df = pypuck.player_stats(start_date, end_date)
```

Comment:
```suggestion
	for start_date_run in [start_date, None]:
		df = pypuck.player_stats(start_date_run, end_date)
```
Same reasoning for this one as for lines 25-26.

Reply:
This one doesn't need to be overwritten, but it's inconsequential that it is being overwritten. Because I'd like to keep the above line the same, for consistency purposes I would like to keep this line as is too.